L1Trigger/TrackerTFP/interface/DataFormats.h

```diff
@@ -0,0 +1,426 @@
+#ifndef L1Trigger_TrackerTFP_DataFormats_h
+#define L1Trigger_TrackerTFP_DataFormats_h
+
```

Comment:
Each header file should usually start with a comment explaining what they it is for. As this file defines multiple classes, and uses some very dense C++, this is particularly important. Each non-trivial class should also have a comment at the beginning explaining its purpose.

Reply:
done

Reply:
DataFormats.h, as it violates the normal CMSSW convention of only one class per file, also needs a comment at the beginning stating that it defines classes "Process, DataFormat, Format ..."